backend/game/cards/discard_cards/multi_manipulation_cards/test_gas_refinery.py

```diff
@@ -14,4 +14,4 @@ def test_gas_refinery_adds_2_victory_points_to_current_player(player, five_playe
 def test_gas_refinery_subtracts_3_health_from_other_players(player, five_players):
     GasRefinery().immediate_effect(player, five_players)
     assert all(other_players.current_health ==
-               7 for other_players in five_players)
\ No newline at end of file
+               7 for other_players in five_players)
```

Comment:
This should be `constants.DEFAULT_HEALTH - 3` instead of the int 7

Reply:
change made and pushed to recent commit